src/sentry/event_manager.py

```diff
@@ -848,6 +851,21 @@ def _save_aggregate(self, event, hashes, release, **kwargs):
                 existing_group_id = h.group_id
                 break
             if h.group_tombstone_id is not None:
+                key = get_raw_cache_key(project.id, event.event_id)
```

Comment:
This block seems worth factoring into its own function since it's all pretty self-contained. It might fit better in `sentry.filters.preprocess_hashes`